src/renderers/webgl/WebGLState.js

```diff
@@ -525,80 +525,123 @@ function WebGLState( gl, extensions, utils ) {
 
 	function setBlending( blending, blendEquation, blendSrc, blendDst, blendEquationAlpha, blendSrcAlpha, blendDstAlpha, premultipliedAlpha ) {
 
-		if ( blending !== NoBlending ) {
+		if ( blending === NoBlending ) {
```

Comment:
I would write the first block like this. It's a bit easier to read since all operations related to `NoBlending` are in a single `if/else` block.
```js
if ( blending === NoBlending ) {

	if ( currentBlending !== NoBlending ) {

		disable( gl.BLEND );
		currentBlending = NoBlending;

	}

} else {

	if ( currentBlending === NoBlending ) {

		enable( gl.BLEND );

	}

}
```

Reply:
Done!

Reply:
Undone. It's all different now...